.github/workflows/audit-security.yml

```
@@ -0,0 +1,48 @@
+# Copyright 2021 The Engula Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Audit Security
+
+on:
+  push:
+    paths:
+      - '**/**/Cargo.toml'
+  pull_request:
+    paths:
+      - '**/**/Cargo.toml'
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  audit-security:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'audit skip')"
```

```suggestion
    if: "!contains(github.event.head_commit.message, 'skip audit')"
```

Is there an idiom with `'audit skip'`? GitHub uses `'skip ci'` to skip CI and I'd like to follow the pattern.

---

ok